engine/src/player-platform.cpp

```diff
@@ -1876,6 +1876,9 @@ void MCPlayer::setenabledtracks(uindex_t p_count, uint32_t *p_tracks_id)
 				t_enabled = false;
 				MCPlatformSetPlayerTrackProperty(m_platform_player, i, kMCPlatformPlayerTrackPropertyEnabled, kMCPlatformPropertyTypeBool, &t_enabled);
 			}
+            
+            if (p_tracks_id == nil)
+                return;
 			
             for (uindex_t i = 0; i < t_track_count; i++)
```

Comment:
@livecodepanos this line should read `for (uindex_t i = 0; i < p_count; i++)`

Reply:
Thanks @livecodeian, I have update the PR